src/explore-education-statistics-frontend/src/modules/find-statistics/components/Filters.tsx

```diff
@@ -17,45 +16,25 @@ export type FilterChangeHandler = ({
 
 interface Props {
   releaseType?: ReleaseType;
-  showMobileFilters: boolean;
   themeId?: string;
   themes: ThemeSummary[];
-  totalResults?: number;
   onChange: FilterChangeHandler;
-  onCloseMobileFilters: () => void;
 }
 
-const Filters = ({
-  releaseType,
-  showMobileFilters = false,
-  themeId,
-  themes,
-  totalResults,
-  onChange,
-  onCloseMobileFilters,
-}: Props) => {
+const Filters = ({ releaseType, themeId, themes, onChange }: Props) => {
   const { isMedia: isMobileMedia } = useMobileMedia();
-  return (
-    <>
-      {!isMobileMedia && (
-        <div className="govuk-!-margin-top-3">
-          <h2 className="govuk-visually-hidden">Filter publications</h2>
-          <ThemeFilters themeId={themeId} themes={themes} onChange={onChange} />
-          <AdvancedFilters releaseType={releaseType} onChange={onChange} />
-        </div>
-      )}
-      {isMobileMedia && showMobileFilters && (
-        <MobileFilters
-          releaseType={releaseType}
-          themeId={themeId}
-          themes={themes}
-          totalResults={totalResults}
-          onChange={onChange}
-          onClose={onCloseMobileFilters}
-        />
-      )}
-    </>
-  );
+
+  if (!isMobileMedia) {
```

Comment:
Given that the rendering of `FiltersMobile` / `MobileFilters` is controlled by the media query hook in `FindStatisticsPage`, would suggest we also do this for this component as well to be consistent i.e.

```tsx
{isMobileMedia && (
  <FiltersDesktop ... />
)}
```

Also means we have one less media query listener.